File: shopica-client/src/app/core/model/comment/comment.ts
export interface Comment {
  id?: number;
  content: string;
  customerId: number;
  email: string;
  customerName: string;
  customerImageUrl: string;
  createdAt?: string;
  rate: number;
  documentType: CommentDocumentType;
  documentId: number;
}

export enum CommentDocumentType {
  Product,
  Blog
}

File: shopica-services/src/Services/Ratting/Ratting.API/Models/Comment.cs
using Ratting.API.Models.Enums;

namespace Ratting.API.Models
{
    public class Comment : BaseEntity
    {
        public int Id { get; set; }
        public string Content { get; set; }
        public int CustomerId { get; set; }
        public string Email { get; set; }
        public string CustomerName { get; set; }
        public string CustomerImageUrl { get; set; }
        public int Rate { get; set; }
        public CommentDocumentType DocumentType { get; set; }
        public int DocumentId { get; set; }
        public IList<UserAction> Actions { get; set; }
    }
}
